libraries/ESP8266WiFi/src/BearSSLHelpers.cpp

```diff
@@ -901,3 +905,17 @@ make_stack_thunk(br_ssl_engine_sendrec_buf);
 #endif
 
 };
+
+#if ARDUINO_SIGNING
+namespace {
+  static BearSSL::PublicKey signPubKey(signing_pubkey);
+  static BearSSL::HashSHA256 __hash;
+  static BearSSL::SigningVerifier __sign(&signPubKey);
+};
+
+namespace esp8266 {
+  UpdaterHashClass& hash = __hash;
```

Comment:
I'll leave the "access yyyy as unnamed namespace __xxxx" stuff up to @devyte to comment on (for me, I'm indifferent to the separation).

However, can we make these var names something more descriptive?  `updaterHash` instead of plain `hash`?  Since this is the only use of the vars, would make it cleaner when we go back and change things again in a couple years. :)

Reply:
Sure, but in the interest of minimal changesets, I haven't touched the external names - except the namespace esp8266, a symbol called hash was just too much of an invitation to name clash disaster :-)

Reply:
Done:
hash -> updaterSigningHash
sign -> updaterSigningVerifier